homeassistant/components/cover/rflink.py

```diff
@@ -105,7 +105,12 @@ def should_poll(self):
     @property
     def is_closed(self):
         """Return if the cover is closed."""
-        return None
+        return not self._state
+
+    @property
+    def assumed_state(self):
+        """Return True because covers can be stopped midway."""
+        return True
```

Comment:
Why is this a property if it returns a constant?

Reply:
I know that the asociated PR is closed, but IMHO 'stop' command must also set the cover state to TRUE.
In others words, if you stop the cover halfway, the cover will stay open.
```
        command = event['command']
        if command in ['on', 'allon', 'up', 'stop']:
            self._state = True
        elif command in ['off', 'alloff', 'down']:
            self._state = False
````
(sorry if it's not the correct place to reply)